deprecated_ros1_pkgs/controller_stopper/package.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
```

Comment:
For these packages in /deprecated_ros1_pkgs, I wasn't planning to build them. They don't seem necessary in the short term.

Just a note so you don't spend time on this deprecated folder in the future.  (Maybe we should just delete the folder?)

Reply:
Ha ha, with my emacs macros, it's actually more work to make exceptions!

The `ur_robot_driver/launch/ur_control.launch` file still refers to `controller_stopper`.  I'll just remove `ur_calibration` for now.